e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+create_infra "$namespace"
+cluster="some-name"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running "$cluster-rs0" 3
+
+check_service() {
+	kubectl_bin patch psmdb "$cluster" --type=json --patch '[
+        {
```

Comment:
I believe it would be better start with adding custom annotations to service and than add values to ignore. We need to test if don't do any changes in CR that operator does not control custom annotations/labels.

Reply:
we can check this task for details https://jira.percona.com/browse/K8SPS-228